### DISCUSSION TITLE
Recovery on private menu

### DISCUSSION THREAD

**Discussion by SlavicBlin** [tupoy-ya/TupoyeMenu#67] [Q&A]
i know this menu is not about recovery or anything but i just wanted to know if
anyone knows how to add something like Rig Casino Slot to the menu privatly ( Only have it on your menu ) or if anyone has a fork with that already in it.

again i know this is not what the menus about but i really cant figure out how to add this.

**Comment by tupoy-ya** [comment]
Personally i don't know how to implement recovery features as i have not researched them and not interested in doing so.

I have seen some forks with pasted recovery somewhere on GitHub, but i don't usually pay attention to those and have no idea about the safety of their methods.

There may be some documentation about rigging casino slots on UnknownCheats, at least for older versions, you will just need to port it to YimMenu and update it.

**Reply by SlavicBlin** [reply]
yh i was looking around on UnknownCheats and found old methods and C# methods of implementing them, i have also found the decompiled scripts for it, but as of now i have only found one person who got it to work but they are not responding unfortunatly. 

as always thx for the help and quick responsed 👍
